extensions/wikia/FounderProgressBar/FounderProgressBarController.class.php

```diff
@@ -37,7 +37,6 @@ class FounderProgressBarController extends WikiaController {
 		'FT_RECENTCHANGES_VISIT' => 230,
 		'FT_WORDMARK_EDIT' => 240,
 		'FT_MOSTVISITED_VISIT' => 250,
-		'FT_TOPTENLIST_ADD' => 260,
 		'FT_BLOGPOST_ADD' => 270,
 		'FT_FB_LIKES_3' => 280,
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "FT_FB_LIKES_3" 4 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1192)